styles/components/header.module.scss

```diff
@@ -14,15 +15,27 @@
 		color: colors.$secondary;
 		font-size: min(4vw, 3rem);
 	}
+
+	a {
+		text-decoration: underline;
+		text-decoration-thickness: min(0.2vw, 0.075rem);
```

Comment:
There is no point in making the underline responsive - at its start its very thin, the changes in sizes are negligible

Reply:
noted